iroha_client_no_std/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod client;
+pub mod model;
```

Comment:
We don't need model module.

Reply:
@humb1t why not? I think it is better to keep all entities in its own place.

Reply:
In Java world especially exist tendency to place non domain aggregations (model, repository, service) on top of the packages structure.
In Iroha we move towards another approach where modules relate to the domain - that's why we do not have such aggregates. You can just move account, domain and other modules on top of the crate.